clone.js

```diff
@@ -0,0 +1,104 @@
+const readline = require('readline').createInterface({
```

Comment:
wat doet deze file?

Reply:
Check issue #33 van vl-blueprint. Ik verwijder de clone.js.

Reply:
Als je een antwoord wilt op je vraag, @cambiph to the rascue.